src/Npgsql/Compatibility/CrateDB/CrateDBDatabaseInfo.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Npgsql.TypeMapping;
+using NpgsqlTypes;
+using System.Data;
+using Npgsql.TypeHandlers;
+
+namespace Npgsql.Compatibility.CrateDB
+{
+    class CrateDBDatabaseInfo : DatabaseInfo
```

Comment:
Could you please explain why a CreateDB-specific DatabaseInfo exists? Does CrateDB not support the regular type loading query, and if so, which part of the catalog schema isn't supported? Or does this exist mainly in order to add the VendorSpecificTypeMappings?